### PR TITLE
DOCS-1943 Rework TrueCommand 2 Install Docs for VM deploy script

### PR DESCRIPTION
**Pull request by tonyriv3**

### CODE REVIEW

File: content/TrueCommand/TCGettingStarted/Install/VMDeploy.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+---
+title: "VM Deployment"
+weight: 40
```
Comment (Mrt134) [inline content/TrueCommand/TCGettingStarted/Install/VMDeploy.md:3]: Ensure article is first in the list of "Installing" subtopics
```suggestion
weight: 10
```

**Comment by Mrt134** [comment]
Added suggestion to fix the weight on the new article, but the file moves are otherwise approved.

File: content/TrueCommand/TCGettingStarted/Install/VMDeploy.md (hunk 1 above)
Comment (tonyriv3) [reply content/TrueCommand/TCGettingStarted/Install/VMDeploy.md]: Alright. I just chose 40 because the others were 50 and 60. Fixed!

**Review by ericbsd — APPROVED** [review]
LGTM

**Comment by Mrt134** [comment]
One false positive with the Beta docs link, ok to merge.

**Pull request merged**